src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,91 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+// import java.util.Date;
+import java.time.LocalDate;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
- * Date Modified: 10/03/17
+ * Date Modified: 10/20/17
  */
 
-public class Meal implements DietFacts {
+public class Meal implements DietFacts{
+
+      //date
+    private LocalDate date;
+    private String title;
+    //foodList
+    private ArrayList <FoodEntry> foodList;
+    private String MealTitle;
+    private int MealCalories;
+    private int MealFat;
+    private int MealCarb;
+    private int MealProtein;
+
```

Comment:
You should not have derived data fields - e.g. data fields that you can calculate from the information of other data fields. Since we can calculate `mealCalories` (and following data fields) form the information in the `foodList`, you should not have these data fields:
+    private int MealCalories;
 +    private int MealFat;
 +    private int MealCarb;
 +    private int MealProtein;

These are needed to be removed.

Instead you should have methods that return these values, which we have via interface.
Also `private String MealTitle;` is redundant and should be removed.